mvcc/backend/batch_tx.go

```diff
@@ -323,6 +319,17 @@ func (t *batchTxBuffered) unsafeCommit(stop bool) {
 	}
 }
 
+func waitAndRollback(tx *bolt.Tx, wg *sync.WaitGroup, lg *zap.Logger) {
+	wg.Wait()
```

Comment:
we need to create a metric for this. make sure there is no leaky transactions.

Reply:
Agreed. Thanks!

Reply:
this is important since a leaky read tx will keep boltdb size growing.

Reply:
got it.